### PR TITLE
BaSP-TG-32: Modal component created and styled

### PR DESCRIPTION
**Pull request by FedeBrus64**
This component works the following way:
- First, it receives a function, called handleClose(), you must pass it a boolean to toggle the modal show/hide
- A boolean variable, isOpen. If true, the modal will show
- Children, which is the internal part of the modal. Here is where you put everything the modal will show (I did not put accept/cancel buttons, since the modal could be for confirming an action or just notifying something)

You can use this in your index for it to show:
- First, paste this in the declarations: const [isAdding, setIsAdding] = useState(false);
- Then, in the return part of the function in your index file, paste this:
<Modal
        handleClose={() => {
          setIsAdding(false);
        }}
        isOpen={isAdding}
        isConfirmation={false}
      >
        children
      </Modal>
- Replace 'children' with what you want the modal to show.

The styles have been based on the figma prototype.

I have tested it and it works. Still, let me know if anything happens.

### CODE REVIEW

File: src/Components/Shared/Select/Select.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+            <label className="">
+                {text}
+            </label>
+            <select className="" id={id} value={value} required={required}>
+            </select>
+        </div>
+    );
+};
+
+export default Select;
```
Comment (lautaro23) [inline src/Components/Shared/Select/Select.jsx:16]: agregar un espacio despues del export default.

**Review by lautaro23 — APPROVED** [review]

File: src/Components/Shared/Select/Select.jsx (hunk 1 above)
Comment (FedeBrus64) [reply src/Components/Shared/Select/Select.jsx]: Corrected!

File: src/Components/Shared/Modal/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          {children}
+          <div className={styles.buttonwrapper}>
+            <button className={styles.modalButton} onClick={handleClose}>
+              OK
+            </button>
+          </div>
+        </div>
+      </div>
+    );
+  }
```
Comment (BaltazarCortina1) [inline src/Components/Shared/Modal/Modal.jsx:46]: Esto es practicamente el mismo codigo en ambos casos. Se podria dejar una sola vez y agregar solo lo que falta cuando `isConfirmation` sea `true`.

**Review by BaltazarCortina1 — CHANGES_REQUESTED** [review]

**Review by Julian25 — APPROVED** [review]

**Review by CorinaArreguez — APPROVED** [review]

**Review by BaltazarCortina1 — APPROVED** [review]

File: src/Components/Shared/Modal/Index.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          {children}
+          <div className={styles.buttonwrapper}>
+            <button className={styles.modalButton} onClick={handleClose}>
+              OK
+            </button>
+          </div>
+        </div>
+      </div>
+    );
+  }
```
Comment (BaltazarCortina1) [inline src/Components/Shared/Modal/Index.jsx:46]: Esto es practicamente el mismo codigo. Deberia estar solo una vez, y cambiar los botones en funcion de si isConfirmation es true o no

File: src/Components/Shared/Modal/Index.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (BaltazarCortina1) [inline src/Components/Shared/Modal/Index.jsx:1]: El nombre de este archivo deberia ser `index.jsx`, con minuscula

**Pull request merged**